go/carmen/property.go

```diff
@@ -27,6 +27,8 @@ const (
 	// ArchiveCache is a configuration property defining an approximate upper
 	// limit for the in-memory node-cache size of the Archive in bytes.
 	ArchiveCache = Property("ArchiveCache")
+	// StorageCache is the size of cache used for account storages.
```

Comment:
Not sure an external user understands this parameter based on this line of documentation. Please improve.

Reply:
ok, extended